sparse/layout.go

```diff
@@ -122,7 +121,8 @@ func RetrieveLayoutStream(abortStream <-chan error, file *os.File, r Interval, l
 		}
```

Comment:
maybe not, this is only used by tests created not by me, I will just leave it this way for now as long as tests are working for now.